apps/examples/src/examples/state-store/StateStoreExample.tsx

```diff
@@ -0,0 +1,99 @@
+import { TLComponents, Tldraw, TLShape, track, useEditor, useQuickReactor, useValue } from 'tldraw'
+import 'tldraw/tldraw.css'
+
+// [1]
+const InfoPanel = track(() => {
+	const editor = useEditor()
+	const tool = editor.getCurrentToolId()
+	const zoom = editor.getZoomLevel().toFixed(2)
+	useQuickReactor(
+		'only red shapes',
+		() => {
+			const shapes = editor.getCurrentPageShapes()
+
+			editor.updateShapes(
+				shapes.map((shape: TLShape) => {
+					if ('color' in shape.props) {
+						return { ...shape, props: { ...shape.props, color: 'red' } }
+					} else {
+						return shape
+					}
+				})
+			)
+		},
+		[editor]
+	)
```

Comment:
this isn't a great example of `useQuickReactor`. This kind of thing would typically be done with the store sideEffects api.

I think a better option would be updating `document.title` to say how many shapes there are on the canvas. I also think the example should be for regular old `useReactor` instead of `useQuickReactor` which is only really useful for us to avoid tearing while things move around.